extensions/azure/cosmos/cosmos-common/src/test/java/org/eclipse/dataspaceconnector/cosmos/azure/CosmosDbApiImplIntegrationTest.java

```diff
@@ -132,4 +134,25 @@ void queryItems() {
         assertThat(result).hasSize(2);
     }
 
+    @Test
+    void deleteItem_whenItemPresent_deleted() {
+        var testItem = new TestCosmosDocument("payload", PARTITION_KEY);
+        container.createItem(testItem);
+        var deletedItem = cosmosDbApi.deleteItem(testItem.getId());
+
+        assertThat(deletedItem).isNotNull().isInstanceOf(LinkedHashMap.class);
+
+        assertThat((LinkedHashMap) deletedItem).containsEntry("id", testItem.getId())
```

Comment:
combine with the above to avoid a second type check/cast.

```
assertThat(queryResult).isNotNull()
                .asInstanceOf(MAP)
                .containsEntry("id", testItem.getId())
                .containsEntry("partitionKey", PARTITION_KEY)
                .containsEntry("wrappedInstance", "payload");
```

from `InstanceOfAssertFactories`

Reply:
Thanks a lot! I also applied this way to previous tests then.